root/programs/C#/Frameworks/Infrastructure/ServiceInterface/AsyncProcessingService/TMInProcessDefinition.xml

```diff
@@ -6,5 +6,5 @@
 ]>
 <!-- idの先頭には、数字を使用できない。 -->
 <TMD>
-  <Transmission id="AzureStrageCopy" assemblyName="AsyncProcess" className="AsyncProcess.ProcessCopyJob" />
+  <Transmission id="AAA" assemblyName="AsyncSvc_sample" className="AsyncSvc_sample.LayerB" />
```

Comment:
Sure, I think there is room for improvement.
Since not included in the contents of this request,
I suggest that you register to issue as backlog.